components/script/webdriver_handlers.rs

```diff
@@ -356,6 +356,35 @@ pub fn handle_add_cookie(
         .unwrap();
 }
 
+pub fn handle_delete_cookies(
+    documents: &Documents,
+    pipeline: PipelineId,
+    reply: IpcSender<Result<(), WebDriverCookieError>>,
+) {
+    let document = match documents.find_document(pipeline) {
+        Some(document) => document,
+        None => {
+            return reply
+                .send(Err(WebDriverCookieError::UnableToDeleteCookies))
+                .unwrap();
+        },
+    };
+    reply
+        .send(
+            match document
+                .window()
+                .upcast::<GlobalScope>()
+                .resource_threads()
+                .send(DeleteCookies())
+                .unwrap()
+            {
+                () => Ok(()),
+                //Used match to add Error Handling Cases later on
```

Comment:
I'm not sure I understand what this comment means. The only thing an error here tells us is that we could note send a message to the resource thread.